ynput/ayon-ftrack
# AYON to ftrack sync: Sync AYON changes to ftrack

## Changelog Description
Synchronization of AYON changes to ftrack.

## Additional review information
This PR combination of multiple other PRs.
- New service 'transmitter' in ftrack addon which listens to AYON events and handles sync to ftrack.
- Synchronization of assignees between tasks if ftrack id is filled in AYON.
- Synchronization of statuses from AYON to ftrack.
- Synchronization of attributes from AYON to ftrack.
- Simple synchronization of comment from AYON to ftrack. We don't capture removed comments in AYON (future enhancement).

## Testing notes:
1. Clone repository and go to `./service_tools`.
2. Run `./manage.ps1 install` on windows or `make install` on linux/mac.
3. Run `./manage.ps1 services` on windows or `make services` on linux/mac (You can run just `transmitter` instead of `services`).
4. Changes in AYON should propagate to ftrack and changes in ftrack should propagate to AYON.

Resolves https://github.com/ynput/ayon-ftrack/issues/133
